safe-svg.php

```diff
@@ -627,11 +634,21 @@ protected function svg_dimensions( $svg ) {
 				}
 			}
 
-			return array(
+			$dimensions = array(
 				'width'       => $width,
 				'height'      => $height,
 				'orientation' => ( $width > $height ) ? 'landscape' : 'portrait',
 			);
+
+			/**
+			 * Calculate SVG dimensions and orientation.
+			 *
+			 * @param array  $dimensions An array containing width, height, and orientation.
+			 * @param string $svg        The file path to the SVG.
+			 *
+			 * @return array An array of SVG dimensions and orientation.
+			 */
+			return apply_filters( 'safe_svg_dimensions', $dimensions, $svg );
```

Comment:
@sksaju @10up/open-source-practice I like this filter, but what are your thoughts on introducing a `safe_svg_pre_dimensions` filter also? 

My thinking is, if someone wants to override this value, forcing them through the overhead of loading and sizing the SVG seems a little odd.

Something like the following at the top of this method to allow the short-circuit might be useful:

```php
/**
* Calculate SVG dimensions and orientation.
*
* This filter allows you to implement your own sizing. By returning a non-false
* value, it will short-circuit this function and return your set value.
*
* @param array  $attachment_id The attachment ID of the SVG being processed.
*
* @return array|false An array of SVG dimensions and orientation or false.
*/
$short_circuit = apply_filters( 'safe_svg_pre_dimensions', '__return_false' );

if ( false !== $short_circuit ) {
    return $short_circuit;
}

```

Reply:
Thanks @darylldoyle I've updated the PR, please take a look again and let me know your feedback. could you also let me know how to fix this https://github.com/10up/safe-svg/actions/runs/7004501147/job/19052370329?pr=154 UnitTest issue? Thanks